crates/volta-core/src/tool/package_global/uninstall.rs

```diff
@@ -21,25 +22,30 @@ pub fn uninstall(name: &str) -> Fallible<()> {
 
     // If the package config file exists, use that to remove any installed bins and shims
     let package_config_file = home.default_package_config_file(name);
-    let package_found = if package_config_file.exists() {
-        let package_config = PackageConfig::from_file(&package_config_file)?;
 
-        for bin_name in package_config.bins {
-            remove_config_and_shim(&bin_name, name)?;
+    let package_found = match PackageConfig::from_file(&package_config_file) {
```

Comment:
This I think would be simplified by a `PackageConfig::from_file_if_exists` as well.

Reply:
Done